common/os/packagemanager.go

```diff
@@ -0,0 +1,16 @@
+package os
+
+import (
+	"github.com/DataDog/test-infra-definitions/command"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+func NewAptManager(runner *command.Runner) command.PackageManager {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Why some managers are here and some in `aws/scenarios/vm/os/packagemanager.go` ? Are these available in both aws and azure ?

Reply:
`common/os` defines `Ubuntu` and `Windows` OSes as they will be available for all datacenters. I kept the package managers in the same go package. For OSes only supported by AWS, the package manager are defined in aws package.